web/app/themes/dp-timber-theme/components/molecule/Card/card.twig

```diff
@@ -0,0 +1,19 @@
+<div class="card-container mx-auto mt-3">
+	<div class="card text-center">
+		<div class="card-header">
+			Storybook Card
+		</div>
+		<div class="card-body">
+			{% if cardImage %}
+				{% include '@atoms/image/image.twig' with { src: cardImage, alt: 'Logo', classes: 'm-3 shadow-sm rounded', width: '240px', height: '' } %}
+			{% endif %}
+			<h5 class="card-title">{{ cardTitle }}</h5>
```

Comment:
render all the fields on a conditional basis. without content, the wrapper should also not render.

Reply:
Okay I will update those changes